### ISSUE TITLE
Create and API for this app?!

### ISSUE THREAD

**Issue by sunscreem** [sunscreem/dogpubtimemachine#13]
I've had a request that the data gathered here is available externally. This is getting a bit meta now.

**Comment by b099l3** [comment]
Yeah I could make a mobile App if the data is exposed in an API of some sort.

**Comment by sunscreem** [comment]
That would be great. I'll probably leave that side of things to @billythekid (since he requested it)... I'll focus on getting the main app built for now
